frontend/package.json

```diff
@@ -21,23 +21,23 @@
     "build:analyze": "export _ODH_OUTPUT_ONLY=true; run-s build build:bundle-profile build:bundle-analyze",
     "build:bundle-profile": "webpack --config ./config/webpack.prod.js --profile --json > ./bundle.stats.json",
     "build:bundle-analyze": "webpack-bundle-analyzer ./bundle.stats.json",
-    "build:storybook": "build-storybook -s ./config",
+    "build:storybook": "storybook build -s ./config",
     "build:clean": "rimraf ./public",
     "build:prod": "webpack --config ./config/webpack.prod.js",
     "format": "prettier --write \"./**/*.ts\" \"./**/*.tsx\" \"./**/*.scss\"",
     "start:dev": "webpack serve --hot --color --config ./config/webpack.dev.js",
-    "test": "run-s test:lint test:jest test:type-check",
-    "test:jest": "jest",
-    "test:ci": "export CI=true; jest --collectCoverage",
-    "test:clearCache": "jest --clearCache",
-    "test:dev": "run-s test:lint test:local",
+    "test": "run-s test:lint test:unit test:type-check test:integration-ci",
+    "test:dev": "run-s test:lint test:unit test:accessibility test:integration",
     "test:fix": "eslint --ext .json,.js,.ts,.jsx,.tsx ./src --fix",
     "test:lint": "eslint --max-warnings 0 --ext .json,.js,.ts,.jsx,.tsx ./src",
-    "test:local": "jest --watchAll",
     "test:type-check": "tsc --noEmit",
-    "test:storybook": "test-storybook --verbose",
-    "test:storybook-ci": "npx concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:storybook\"",
-    "storybook": "start-storybook -p 6006 -s ./config"
+    "test:jest": "jest",
+    "test:unit": "npm run test:jest",
+    "test:accessibility": "test-storybook --verbose",
+    "test:integration": "playwright test ./src/__tests__/integration",
+    "test:integration-ci": "npx concurrently -k -s first -n \"SB,TEST\" \"npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent\" \"npx wait-on tcp:127.0.0.1:6006 && npm run test:accessibility && npm run test:integration\"",
```

Comment:
Running `npm run test` is now not completable... Why did we bring a CI element into a command line argument for testing?

Reply:
what do you mean? does `npm run test` not work for you? should i have the `test` not build storybook and instead have a `test-ci`

Reply:
Oh interesting -- maybe because I didn't build storybook? I'll give that a shot... it never completes when I run test, server running or not.

Reply:
```
[TEST] npx wait-on tcp:127.0.0.1:6006 && npm run test:accessibility && npm run test:integration exited with code 0
--> Sending SIGTERM to other processes..
[SB] npm run build:storybook --quiet && npx http-server storybook-static --port 6006 --silent exited with code SIGTERM
```
@Gkrumbach07 this is what I get when running `npm run test`

Reply:
it should be running a bunch of other tests too. those are the only lines its putting out?

Reply:
Those are the last ones 😞 -- before that is it executing the previous test

Reply:
i think i resolved the issue. i was searching for a button with the text "Create", however this was searching for a non exact match so it was finding other buttons with the word create in it. I am not sure why it was passing for me locally and in CI for some of the runs